packages/testkit-backend/src/request-handlers.js

```diff
@@ -43,16 +42,18 @@ export function NewDriver (context, data, wire) {
   }
   const resolver = resolverRegistered
     ? address =>
-      new Promise((resolve, reject) => {
-        const id = context.addResolverRequest(resolve, reject)
-        wire.writeResponse(responses.ResolverResolutionRequired({ id, address }))
-      })
+        new Promise((resolve, reject) => {
+          const id = context.addResolverRequest(resolve, reject)
+          wire.writeResponse(responses.ResolverResolutionRequired({ id, address }))
+        })
```

Comment:
:eyes: was that on purpose?

Reply:
I've reformatted the file, so it might be the correct indentation.